docs/guides/google-auth-hook.md

```diff
@@ -253,7 +254,7 @@ const options = {
 };
 
 if (process.env.DATABASE_URL_FILE) {
-  options.databaseUrl = fs.readFileSync(process.env.DATABASE_URL_FILE);
+  options.databaseUrl = fs.readFileSync(process.env.DATABASE_URL_FILE, 'utf8');
```

Comment:
We can now remove the need for this in the guide as it is now done in core automatically.